Vagrantfile

```diff
@@ -56,6 +70,10 @@ Vagrant.configure("2") do |config|
   # and docker or gunicorn (8000) to host machine
   config.vm.network "forwarded_port", guest: 4000, host: 4000
   config.vm.network "forwarded_port", guest: 8000, host: 8000
+  config.vm.synced_folder ".", "/vagrant"
+  
+  # Fix ssh problem
+  config.vm.provision :shell, :inline => "sudo sed -i 's/PasswordAuthentication no/PasswordAuthentication yes/g' /etc/ssh/sshd_config; sudo systemctl restart sshd;", run: "always"
```

Comment:
I feel like this shouldn't be necessary. Don't most vagrant images come pre-installed with vagrant's default SSH key?